Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+
+class TransformStack: NSObject {
+    
+    static var shared: TransformStack = TransformStack()
+    
+    weak var transformDelegate : TransformDelegate!
+    
+    var transformAdjustmentsStack : Array = [TransformRecord]()
+    
+    var isRotated : Bool! = false
+    
+    var bottom : Int = 0
+    var top : Int = 0
+    
+    var transformStackBottom : Int = 0
+    var transformStackTop : Int = 0
+        
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override init() {
```

Comment:
I think you can add `private` here since TransformStack is Singleton

Reply:
changes made